Updated KFC entry with alt_name:vi as 'Gà Rán Kentucky

ditrungduong opened the pull request:
- Updated KFC entry in `fast_food.json`
- Added `alt_name:vi` as "Gà Rán Kentucky" while keeping `name:vi` as "KFC"
- This follows the localization guidelines and maintains consistency with branding
- Related to Issue #10553

Snowysauce commented:
The build tests failed because "(Vietnam)" (or the appropriate localization thereof) needs to be added to the end of the new entry's display name.

Other suggestions:

- `brand:vi` is unnecessary since its value is identical to `brand`
- Vietnam needs to be excluded from the global KFC entry; otherwise, Vietnamese editors will be given an option to use this proposed preset or the global one

ditrungduong commented:
Thanks for the feedback! I’ve made the following corrections based on your suggestions:

Updated displayName to "KFC (Vietnam)" to meet localization requirements.
Removed brand:vi since it was redundant.
Updated the global KFC entry to exclude "vn" so that Vietnamese editors will only see the localized option.
Please let me know if any further changes are needed!
